assignments/assignment-6/public/scripts.js

```diff
@@ -1,8 +1,21 @@
+var currentScoreAmount = 300;
```

Comment:
Why do we have this starting at 300?

Reply:
That was just a test value, the value it's set to doesn't matter.

Reply:
We should prob set this to 0 then, unless we can remove it altogether by refactoring scoring so it doesn't use global vars.